HelpSource/scdoc.js

```diff
@@ -105,7 +105,7 @@ function addInheritedMethods() {
             var m = mets[j];
             if(doc.methods.indexOf("_"+m.slice(1))<0) { // ignore methods already documented in this helpfile
                 var li = document.createElement("li");
-                li.innerHTML = "<a href='"+helpRoot+"/"+s.path+".html#"+m.slice(1)+"'>"+m.slice(2)+"</a>";
+                li.innerHTML = "<a href='"+helpRoot+"/"+s.path+".html#"+encodeURIComponent(m.slice(1))+"'>"+m.slice(2)+"</a>";
```

Comment:
slightly better would be just `href='" + encodeURI(helpRoot + "/" + s.path + ".html#" + m.slice(1) +"'...`

because the path may have spaces in it too.  encodeURI does the right thing and knows about ? and # and all the nasty chars that need to be encoded. http://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_encodeuri